src/components/crossDevice/MobileFlow.js

```diff
@@ -9,6 +9,8 @@ class MobileFlow extends Component {
     this.props.socket.on('disconnect ping', this.onDisconnectPing)
     this.props.socket.on('get config', this.sendConfig)
     this.props.socket.on('client success', this.onClientSuccess)
+
+    addEventListener('userAnalyticsEvent', (event) => this.sendUserAnalyticsEvent(event));
```

Comment:
I left a very long message here but it got lost when I clicked on "Start review". Summary:
This component is part of the desktop flow, despite its unfortunate name, so this won't work. You would need to make sure that the "desktop" SDK is aware that the SDK running on id.onfido.com is calling `sendEvent`. Therefore you will have to do send a message from the sdk running on id.onfido.com (aka cross device client) to desktop by calling `socket.emit(roomId, 'user analytics event', event)` every time the `sendEvent` function is called and `mobileFlow` is `true` (`mobileFlow` is true when the SDK is running inside `id.onfido.com`). The message will have to be handled by the "desktop" SDK which will then call sendEvent again, this time firing the actual event. We can probably agree on a better design for this and if this sounds confusing we can have a call later.